src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -513,7 +520,14 @@ private WizardState getWizardStateFor_(X x, Subject s, String capabilityId) {
       try {
         var ucj = getJunction(x, cap.getId());
         if ( IN(UserCapabilityJunction.STATUS, grantedStatuses).f(ucj) ) {
-          granted.add(cap.getId());
+          // if a minmax capability is in GRANTED status, its prerequisites should also be added
+          // as a part of granted list, so the wizrd will know to ignore the prerequisites
+          if ( isMinmax ) {
```

Comment:
`isMinmax` should be `isPrerequisiteAware` - this condition is true for any capability that has a wizardlet implementing PrerequisiteAware.